content/manuals/subscription/details.md

```diff
@@ -0,0 +1,406 @@
+security across organizations. It enables groups of developers to unlock the
+full power of collaboration and sharing combined with essential security
+features and team management capabilities. A Docker Team subscription includes
+licensing for commercial use of Docker components including Docker Desktop,
+Docker Hub, Docker Scout, Docker Build Cloud, and Testcontainers Cloud.
+
+Docker Team includes:
+
+- 500 Docker Build Cloud minutes per month.
+- Unlimited Docker Scout repositories with continuous vulnerability analysis.
+- 500 Testcontainers Cloud runtime minutes per month for use either in Docker Desktop or for CI.
+- No Docker Hub image pull rate limits.
+- 100K Docker Hub pulls per month.
+
+There are also advanced collaboration and management tools, including
+organization and team management with [Role Based Access Control
+(RBAC)](/security/for-admins/roles-and-permissions/), [activity logs](/admin/organization/activity-logs/), and more.
+
+For a list of features available in each tier, see [Docker
+Pricing](https://www.docker.com/pricing/).
+
+## Docker Business
+
+**Docker Business** offers centralized management and advanced security features
+for enterprises that use Docker at scale. It empowers leaders to manage their
+Docker development environments and speed up their secure software supply chain
+initiatives. A Docker Business subscription includes licensing for commercial use of
+Docker components including Docker Desktop, Docker Hub, Docker Scout, Docker
+Build Cloud, and Testcontainers Cloud.
+
+Docker Business includes:
+
+- 1500 Docker Build Cloud minutes per month.
+- Unlimited Docker Scout repositories with continuous vulnerability analysis.
+- 1500 Testcontainers Cloud runtime minutes per month for use either in Docker Desktop or
+  for CI.
+- No Docker Hub image pull rate limits.
+- 1M Docker Hub pulls per month.
+
+In addition, you gain access to enterprise-grade features, such as:
+- [Hardened Docker Desktop](../security/for-admins/hardened-desktop/_index.md)
+- [Image Access
+  Management](../security/for-admins/hardened-desktop/image-access-management.md)
+  which lets admins control what content developers can access
+- [Registry Access
+  Management](../security/for-admins/hardened-desktop/registry-access-management.md)
+  which lets admins control what registries developers can access
+- [Company layer](/admin/company/) to manage multiple organizations and settings
+- [Single sign-on](/security/for-admins/single-sign-on/)
+- [System for Cross-domain Identity
+  Management](/security/for-admins/provisioning/scim/)
+
+For a list of features available in each tier, see [Docker
+Pricing](https://www.docker.com/pricing/).
+
+### Self-serve
+
+A self-serve Docker Business subscription is where everything is set up by you.
+You can:
+
+- Manage your own invoices
+- Add or remove seats
+- Update billing and payment information
+- Downgrade your subscription at any time
+
+### Sales-assisted
```

Comment:
**suggestion:** The new offering applies sales-assisted to Docker Team in addition to Docker Business. I think both Self-serve and Sales-assisted should be moved out from Docker Business to make them generic.